app/src/main/java/com/sdp13epfl2021/projmag/activities/ProjectInformationActivity.kt

```diff
@@ -88,6 +85,39 @@ class ProjectInformationActivity : AppCompatActivity() {
         }
     }
 
+    private fun setApplyButtonText(applyButton: Button, applied: Boolean) {
+        if (applied)
+            applyButton.text = "UNAPPLY"
+        else
+            applyButton.text = "APPLY"
+    }
+
+    private fun setUpApplyButton(applyButton: Button) {
+        val projectId = projectVar.id
+        val userDataDatabase = Utils.getInstance(this).userDataDatabase
+        var applied = false
+        userDataDatabase.getListOfAppliedToProjects({
+                projectIds ->
+            if (projectIds.contains(projectId))
+                applied = true
+            setApplyButtonText(applyButton, applied)
```

Comment:
One line if without {} can be error-prone, and the indentation seems a little weird for projectIds.
```suggestion
        userDataDatabase.getListOfAppliedToProjects({ projectIds ->
            applied = projectIds.contains(projectId)
            setApplyButtonText(applyButton, applied)
```